### PR TITLE
Feature/avatar

### PR DESCRIPTION
**Pull request by albinr99salt**

### CODE REVIEW

File: frontend/rollup.config.js [hunk 1]
```diff
@@ -80,8 +80,8 @@ export default {
 				SERVER_URL: 'hello',
 				FRONT_URL: 'hello',
 				env: {
-					SERVER_URL: process.env.SERVER_URL,
-					FRONT_PORT: process.env.FRONT_PORT,
+					SERVER_URL: process.env.SERVER_URL || 'http://localhost:666',
```
Comment (albinr99salt) [inline frontend/rollup.config.js:83]: Denna ska bort

File: frontend/src/api-client/posts.ts [hunk 2]
```diff
@@ -24,8 +24,10 @@ export class Post extends Client {
       type: "formData",
     });
 
-    if(!id) {
-      throw new InternalServerError("No id in response")
+    console.log("id", id);
```
Comment (albinr99salt) [inline frontend/src/api-client/posts.ts:27]: Ta bort log

File: server/src/routes/posts.ts [hunk 3]
```diff
@@ -103,7 +63,49 @@ router.post('/create/:id', loginRequired, async (req, res) => {
...
+
+    return res.json(post.id);
+  } catch (error) {
+    return res.json({
+      status: false,
+      message: error,
+    });
+  }
+
+  // const inserts = await Post.create({
```
Comment (albinr99salt) [inline server/src/routes/posts.ts:101]: Delete comments

File: frontend/src/views/Dashboard/Dashboard.svelte [hunk 4]
```diff
@@ -23,6 +25,13 @@
         document.body.style.position = '';
         document.body.style.top = '';
     }
+
+    const uploadAvatar = async () => {
+        console.log(file[0])
```
Comment (albinr99salt) [inline frontend/src/views/Dashboard/Dashboard.svelte:30]: remove logs

File: server/src/server.ts [hunk 5]
```diff
@@ -30,8 +31,19 @@ const server = async () => {
...
+      app.use(
+        cors({
+          origin: 'http://localhost:3000',
+          credentials: true,
+        })
+      );
+      app.use(
+        morgan('combined', {
+          skip: (_, res) => {
+            return res.statusCode ? res.statusCode < 400 : false;
```
Comment (albinr99salt) [inline server/src/server.ts:43]: We should keep 200 logs as well.

File: server/src/server.ts [hunk 6]
```diff
@@ -57,6 +69,16 @@ const server = async () => {
           store: new MemoryStore(), // Use the memory store
         })
       );
+
+      app.use(
+        fileUpload({
+          limits: { fileSize: 50 * 1024 * 1024 },
```
Comment (albinr99salt) [inline server/src/server.ts:75]: Would be nice with a comment here, something like `//50 mb`

File: server/src/util/color-predictor.ts [hunk 7]
```diff
@@ -10,15 +10,24 @@ const predict = async (url: string): Promise<string> => {
     const green: number[] = [];
     const blue: number[] = [];
 
-    const image = await Jimp.read(url);
+    console.log('url: ' + url);
```
Comment (albinr99salt) [inline server/src/util/color-predictor.ts:13]: remove log

File: server/src/util/color-predictor.ts [hunk 8]
```diff
@@ -10,15 +10,24 @@ const predict = async (url: string): Promise<string> => {
...
+
+    const image = await Jimp.read(url, (err, idk) => {
+      if (err) {
+        throw err;
+      }
+      return idk;
```
Comment (albinr99salt) [inline server/src/util/color-predictor.ts:19]: Use a better name for this

File: server/src/util/color-predictor.ts [hunk 9]
```diff
@@ -10,15 +10,24 @@ const predict = async (url: string): Promise<string> => {
...
+    console.log(image.bitmap.width);
 
     image.cover(
       224,
       224,
       Jimp.HORIZONTAL_ALIGN_CENTER | Jimp.VERTICAL_ALIGN_MIDDLE
     );
 
-    console.log(image.getMIME());
+    // console.log(image.getMIME());
```
Comment (albinr99salt) [inline server/src/util/color-predictor.ts:30]: remove this

**Review by axelburling — APPROVED** [review]

**Pull request merged**